app/templates/index.hbs

```diff
@@ -159,7 +159,7 @@
 <section>
   <div class="layout container">
     <div class="lg:col-4 lg:start-2 text-center">
-      <a href="https://emberconf.com" class="es-button">
+      <a href="https://emberconf.com" class="es-button mb-1">
```

Comment:
Shouldn't this use the same strategy as above? the first one has mb-1 and mr-1 instead of the second one having ml-1? 🤔

Reply:
These buttons we're somewhat annoying to get right with just the functional margin classes as they are centered on screen (the others are left aligned). I chose to give the smaller button the margin. It will appear slightly off center but this is the best we can do without custom css or a more "elaborate" flexbox setup.

<img width="318" alt="Screenshot 2020-02-12 at 12 12 29" src="https://user-images.githubusercontent.com/334789/74329894-f4729200-4d90-11ea-86b5-41e19a4e0790.png">

Reply:
Just to clarify: with the mr-1 approach the bigger button would be off center which would be more noticable.

Reply:
so the only way to fix this perfectly would be with the "button group" implementation that we discussed previously 🤔 hmm

I think I'm going to merge this and create an issue to continue this after the release 👍

Reply:
https://github.com/ember-learn/ember-styleguide/issues/283